README.md

```diff
@@ -1,6 +1,6 @@
 <p align="center">
-  <a href="https://www.notionjs.com/">
-    <img alt="Notion" src="https://github.com/notion-cli/notion/blob/master/notion.png?raw=true" width="480">
+  <a href="https://www.volta.sh/">
+    <img alt="Volta" src="https://github.com/volta-cli/volta/blob/master/volta.png?raw=true" width="480">
```

Comment:
Did you rename the `notion.png` image to `volta.png` in the root directory? I couldn't find it in the PR but maybe I overlooked it.

Reply:
I did not – was going to ask you for the relevant asset first, but I guess it's in the www repo?

Reply:
Update: it's a screenshot and I got one to replace the existing one.

Reply:
Done in 6c69bf03. Thank you!